### ISSUE TITLE
Multible OrderItem with same NamedAddress

### ISSUE THREAD

**Issue by xerc** [baltpeter/internetmarke-php#3]
org.xml.sax.SAXParseException; cvc-complex-type.3.2.2: Attribute 'id' is not allowed to appear in element 'ns1:address'

```
  $this->service->checkoutShoppingCartPdf
  (
    $this->token,
    108,// Zweckform 3659 Etiketten 97 x 42.3
    [
      new \baltpeter\Internetmarke\OrderItem
      (
        21,
        null,
        $address_binding,
        new \baltpeter\Internetmarke\Position(1, 1, 1),
        'AddressZone'
      ),
      new \baltpeter\Internetmarke\OrderItem
      (
        21,
        null,
        $address_binding,
        new \baltpeter\Internetmarke\Position(2, 1, 1),
        'AddressZone'
      )
    ],
    (145 * 2)
  );
```

**Comment by xerc** [comment]
> SoapClient makes a reference when you use the same object on multiple XML nodes.

-> provisional we'll use https://github.com/myclabs/DeepCopy

**Comment by baltpeter** [comment]
Hi,

thanks for the report! Unfortunately, I don't have time to look into this right now but feel free to submit a pull request if you find anything.

**Comment by baltpeter** [comment]
I finally got around to fixing this issue with v0.2.0. For anyone interested, you can find a detailed description of the problem and fix in the commit message for 8082188.
